src/systems/filecoin_mining/storage_mining/mining_cycle.md

```diff
@@ -68,23 +68,15 @@ The miner actor has two distinct 'controller' {{<sref app_address "addresses">}}
 
 #### Changing Worker Addresses
 
-Note that any change to worker keys after registration (TODO: spec how this works) must be appropriately delayed in relation to randomness lookback for SEALing data (see [this issue](https://github.com/filecoin-project/specs/issues/415)).
+Note that any change to worker keys after registration must be appropriately delayed in relation to randomness lookback for SEALing data (see [this issue](https://github.com/filecoin-project/specs/issues/415)).
 
 ### Step 1: Committing Sectors
 
-When the miner has completed their first seal, they should post it on-chain using the {{<sref storage_miner_actor>}}'s `ProveCommitSector` function. The miner will need to put up pledge collateral in proportion to the amount of storage they commit on chain. If the miner had zero committed sectors prior to this call, this begins their proving period.
+When the miner has completed their first seal, they should post it on-chain using the {{<sref storage_miner_actor>}}'s `ProveCommitSector` function. The miner will need to put up pledge collateral in proportion to the amount of storage they commit on chain. If the miner had zero committed sectors prior to this call, this begins their proving period and miner gains power for the sector.
 
 You can read more about sectors {{<sref sector "here">}}.
 
-### Step 2: Getting Power
```

Comment:
even if this is removed, there needs to be something tying proving sectors to gaining power...

Reply:
added link to the section where power is discussed.